### PR TITLE
Feat/feature routes

### PR DESCRIPTION
**Pull request by FredrikOseberg**

### CODE REVIEW

File: src/component/application/__tests__/__snapshots__/application-edit-component-test.js.snap [hunk 1]
```diff
@@ -324,7 +324,7 @@ exports[`renders correctly with permissions 1`] = `
                       className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                     >
                       <a
-                        href="/features/strategies/ToggleA"
+                        href="/projects/undefined/toggles/ToggleA/strategies/ToggleA"
```
Comment (ivarconr) [inline src/component/application/__tests__/__snapshots__/application-edit-component-test.js.snap:327]: huh. "undefined"?

File: src/component/application/application-view.jsx [hunk 2]
```diff
@@ -89,18 +89,21 @@ function ApplicationView({
...
                                 show={notFoundListItem({
-                                    createUrl: '/features/create',
+                                    createUrl: `/projects/${project}/toggles`,
                                     name,
                                     permission: CREATE_FEATURE,
                                     i,
                                 })}
                                 elseShow={foundListItem({
-                                    viewUrl: '/features/strategies',
+                                    viewUrl: `/projects/${project}/toggles/${name}/strategies`,
```
Comment (ivarconr) [inline src/component/application/application-view.jsx:106]: we must agree on `toggles` vs `features`. Currently the API will use `features`. I find it weird if we use other urls inside the UI. 

I would prefer the ability to prefix the url wiht "/api/admin" and get the JSON representation.

File: src/component/application/__tests__/__snapshots__/application-edit-component-test.js.snap (hunk 1 above)
Comment (FredrikOseberg) [reply src/component/application/__tests__/__snapshots__/application-edit-component-test.js.snap]: Thanks. Feature toggle data in tests were missing "project" property.

File: src/component/application/application-view.jsx (hunk 2 above)
Comment (FredrikOseberg) [reply src/component/application/application-view.jsx]: Redone to features.

**Review by ivarconr — APPROVED** [review]

**Pull request merged**